index.html

```diff
@@ -4,12 +4,33 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Hello Microverse</title>
+    <title>Setup Mobil Version</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500&display=swap" rel="stylesheet">
 </head>
 <body>
-    <main>
-        <h1 class="greeting">Hello Microverse!</h1>
-    </main>
+  <header>
+    <div class="header__status">
+      <span><img src="./images/time.png" alt="time"></span>
+      <span><img src="./images/status-bar.png" alt="status bar"></span>  
+    </div>
+    <div class="header__links">
+      <a class="logo" href="#"><img src="./icons/logo.png" alt="header icon"></a>
+      <a class="menu" href="#"><img src="./icons/hamburger.png" alt="hamburger menu"></a>
+    </div>  
+  </header>
+  <section class="cover">
+    <h1>I’m Robin<br>Glad to see you!</h1>
+    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <h2 class="cover__subtitle">LET’S CONNECT</h2>
+    <ul class="cover__icons">
+      <li><a href="@MecanicoRobin"><img src="./icons/twitter.svg" alt="twitter"></a></li>
+      <li><a href="https://www.linkedin.com/in/robin-benitez-mora-manizales/"><img src="./icons/linkdin.svg" alt="linkedIn"></a></li>
+      <li><a href="#"><img src="./icons/media.svg" alt="media"></a></li>
+      <li><a href="https://github.com/robinbenitezmora"><img src="./icons/github.svg" alt="github"></a></li>
+      <li><a href="#"><img src="./icons/rabbit.svg" alt="rabbit"></a></li>
+    </ul>
+  </section>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you  also wrap this `<section>` tag inside a `<main>` tag for good HTML semantics. Here is an example of good HTML code:
```
<header>
   code for the header...
</header>

<main>
   <section>
      code for the 1st section...
   </section>
     
   <section>
      code for the 2nd section...
   </section>
   ....
</main>

<footer>
   Code for the footer...
<footer>
```